opensearch/src/http/models.rs

```diff
@@ -0,0 +1,26 @@
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug)]
+#[doc = "Cluster information"]
+pub struct ClusterInfo {
```

Comment:
Let's rename this to `InfoResponse` so it's a bit more obvious what action it relates to.